docs/gui/menus.md

```diff
@@ -272,12 +272,15 @@ A `MenuProvider` is an interface that contains two methods: `#createMenu`, which
 
 A `MenuProvider` can easily be created using `SimpleMenuProvider`, which takes in a method reference to create the server menu and the title of the menu.
 
+##### Open a menu
+
+Use `IForgeServerPlayer#openMenu(MenuProvider, BlockPos)` if you just need to send a pos to create menu.
+Use `IForgeServerPlayer#openMenu(MenuProvider, Consumer<FriendlyByteBuf>)` for more information.(https://github.com/MinecraftForge/MinecraftForge/issues/3228)
+
 ```java
-// In some implementation
-NetworkHooks.openScreen(serverPlayer, new SimpleMenuProvider(
-  (containerId, playerInventory, player) -> new MyMenu(containerId, playerInventory),
-  Component.translatable("menu.title.examplemod.mymenu")
-));
+serverPlayer.openMenu(new SimpleMenuProvider(
+  (containerId, playerInventory, serverPlayer) -> new MyMenu(containerId, playerInventory),
```

Comment:
```suggestion
  (containerId, playerInventory, player) -> new MyMenu(containerId, playerInventory),
```
serverPlayer exists in the outer scope, let's not confuse the reader by overwriting it.